backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/impl/ProductPackingInstructionService.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * #%L
+ * de.metas.handlingunits.base
+ * %%
+ * Copyright (C) 2025 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.handlingunits.impl;
+
+import de.metas.bpartner.BPartnerId;
+import de.metas.product.IProductPackingInstructionService;
+import de.metas.product.ProductId;
+import de.metas.util.Check;
+import lombok.NonNull;
+import org.compiere.model.I_I_Product;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ProductPackingInstructionService implements IProductPackingInstructionService
```

Comment:
i don't think this is the right place, neither the right name for such a service!